src/app/shared/serialization/serializer.service.spec.ts

```diff
@@ -1,12 +1,46 @@
-import { TestBed } from '@angular/core/testing';
+import { TestBed, async } from '@angular/core/testing';
 
 import { SerializerService } from './serializer.service';
 
+import { Base64 } from 'js-base64';
+
 describe('SerializerService', () => {
+
+  let service: SerializerService;
+
+  const testData = 'Test';
+  const jsonData = '"Test"';
```

Comment:
It's strange, I think you should use `toEqual` instead of `toBe` in your test to avoid this "tricks".

The `toBe` is interesting when you want to be sure that your return is of the desired type.

For exemple:
```
notFoundError{
    status: 404,
    error: 'not found'
}
```
Here your error is typed, so it's interesting to use a `toBe` to be sure it's not a simple object, in your case it's not really interesting.